frontend_tests/node_tests/recent_senders.js

```diff
@@ -310,3 +313,30 @@ test("process_message_for_senders", () => {
         true,
     );
 });
+
+test("process_pms", () => {
+    const sender1 = 1; // Current user id
+    const sender2 = 2;
+    const sender3 = 3;
+
+    const user_ids_string = "2,3,4";
+    rs.add_pm_message({
+        to_user_ids: user_ids_string,
+        sender_id: sender2,
+        id: 1,
+    });
+    rs.add_pm_message({
+        to_user_ids: user_ids_string,
+        sender_id: sender3,
+        id: 2,
+    });
+    rs.add_pm_message({
+        to_user_ids: user_ids_string,
+        sender_id: sender1,
+        id: 3,
+    });
+
+    assert.deepEqual(rs.get_pm_recent_senders(user_ids_string), [4, 2, 3, 1]);
```

Comment:
Can you add a comment explaining why this is the answer? I guess mainly the thing I'm confused about is that thi seems like the reverse of the order I'd have expected -- is it oldest first?

Reply:
Did. Recent topics displays them in opposite order, so we need to return the values with lowest message id first.